src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/CborConformanceLevel.cs

```diff
@@ -7,49 +7,55 @@
 
 namespace System.Formats.Cbor
 {
-    /// <summary>
-    ///   Defines supported conformance modes for encoding and decoding CBOR data.
-    /// </summary>
+    /// <summary>Defines supported conformance modes for encoding and decoding CBOR data.</summary>
     public enum CborConformanceMode
     {
-        /// <summary>
-        ///   Ensures that the CBOR data is well-formed, as specified in RFC7049.
-        /// </summary>
+        /// <summary>Ensures that the CBOR data is well-formed, as specified in RFC7049.</summary>
         Lax,
 
-        /// <summary>
-        ///   Ensures that the CBOR data adheres to strict mode, as specified in RFC7049 section 3.10.
-        ///   Extends lax conformance with the following requirements:
-        ///   <list type="bullet">
-        ///   <item>Maps (major type 5) must not contain duplicate keys.</item>
-        ///   <item>Simple values (major type 7) must be encoded as small a possible and exclude the reserved values 24-31.</item>
-        ///   <item>UTF-8 string encodings must be valid.</item>
-        ///   </list>
-        /// </summary>
+        /// <summary><para>Ensures that the CBOR data adheres to strict mode, as specified in RFC7049 section 3.10.</para><para>Extends lax conformance with the following requirements:</para><list type="bullet">
+        /// <item>
+        /// <description>Maps (major type 5) must not contain duplicate keys.</description>
+        /// </item>
+        /// <item>
+        /// <description>Simple values (major type 7) must be encoded as small a possible and exclude the reserved values 24-31.</description>
+        /// </item>
+        /// <item>
+        /// <description>UTF-8 string encodings must be valid.</description>
+        /// </item>
+        /// </list></summary>
         Strict,
 
-        /// <summary>
-        ///   Ensures that the CBOR data is canonical, as specified in RFC7049 section 3.9.
-        ///   Extends strict conformance with the following requirements:
-        ///   <list type="bullet">
-        ///   <item>Integers must be encoded as small as possible.</item>
-        ///   <item>Maps (major type 5) must contain keys sorted by encoding.</item>
-        ///   <item>Indefinite-length items must be made into definite-length items.</item>
-        ///   </list>
-        /// </summary>
+        /// <summary><para>Ensures that the CBOR data is canonical, as specified in RFC7049 section 3.9.</para><para>Extends strict conformance with the following requirements:</para><list type="bullet">
```

Comment:
The indentation of that list element is problematic, but  IIRC we can't influence indentation via System.Xml.Linq?